### ISSUE TITLE
Changing the variable "gender" to "sex" in the esdcomp dataset in the faraway package

### ISSUE THREAD

**Issue by barblu** [julianfaraway/faraway#3]
Dear Julian,

I am working with the _esdcomp_ dataset to show an example to my students. When making the content of my materials gender neutral (or gender inclusive, if you wish), I realised that the _esdcomp_ dataset has the variable "gender" which can take values "M" and "F". I would like to ask if it is possible to change this variable name to "sex", which would be more accurate (indeed, gender cannot assume these values, see https://en.wikipedia.org/wiki/Sex–gender_distinction).

Thanks!
barblu

**Comment by julianfaraway** [comment]
I agree that sex is the appropriate name for this variable according to current terminology. But I am reluctant to change it because this will break scripts written by other users of the package.

**Comment by barblu** [comment]
Thanks for the quick reply.
I understand the reluctance but on the other hand, I think that breaking other users' scripts would be as annoying as it currently is to see the systematic use of wrong terminology, and nothing being done to change that.
